src/BloomBrowserUI/bookEdit/js/bloomImages.js

```diff
@@ -54,38 +54,62 @@ function SetupImage(image) {
     });
 }
 
+function SetButtonModifier(container) {
```

Comment:
Well, it sets the variable that I use later... but you're right. I'll change it to GetButtonModifier. Makes more sense.